omod/src/main/webapp/resources/htmlforms/sierra_leone/section-plan.xml

```diff
@@ -179,288 +179,63 @@
-                    <!-- ToDo: ELISA EID -->
-                    <render labName="PIH:HIV VIRAL LOAD" codeLabel="pihcore.lab.viralLoad" comment="HIV viral load"/>
-                    <render labName="PIH:CD4 COUNT" codeLabel="pihcore.lab.cd4" comment="CD4 count"/>
-                </repeat>
-
-                <!-- Other -->
-                <label>
-                    <uimessage code="pihcore.other"/>
-                </label>
-
-                <includeIf velocityTest="$patient.gender == 'F' ">
-                    <repeat>
-                        <template>
-                            <p>
-                                <obs conceptId="PIH:Lab test ordered coded"
-                                     answerConceptId="{labName}"
-                                     answerCode="{comment}"
-                                     style="checkbox"/>
-                            </p>
-                        </template>
-                        <render labName="CIEL:885" comment="PAP test"/>
-                    </repeat>
-                </includeIf>
-
-                <repeat>
-                    <template>
-                        <p>
-                            <obs conceptId="PIH:Lab test ordered coded"
-                                 answerConceptId="{labName}"
-                                 answerCode="{comment}"
-                                 style="checkbox"/>
-                        </p>
-                    </template>
-                    <render labName="CIEL:302" comment="pihcore.lab.urine"/>
-                    <render labName="CIEL:161454" comment="Gram stain"/>
-                    <render labName="CIEL:163620" comment="H pylori"/>
-                </repeat>
-                <p>
-                    <obs conceptId="PIH:Lab test ordered coded"
-                         answerConceptId="PIH:OTHER"
-                         answerCode="zl.ifOtherSpecify"
-                         style="checkbox"
-                         showCommentField="true" commentFieldLabel=""/>
-                </p>
-            </div>
+            <p>
+                <obs conceptId="PIH:FAMILY PLANNING COMMENT" 
+                labelText="Offered and accepted Family Planning information"  
+                answers="PIH:YES,PIH:NO" 
+                answerLabels="Yes,No" 
+                style="dropdown"
+                answerSeparator=""/>
+            </p>
         </div>
-
     </section>
 
-    <section id="radiology-orders" sectionTag="fieldset" headerTag="legend" headerStyle="title"
-             headerCode="pihcore.rad_order.title">
+      <section id="radiology-ultrasound-orders" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.order.title">
+        <h3>Radiology and ultrasound orders</h3>
+        <div class="section-container-color">
+            <p>
+                <h3>X-ray 1 ordered</h3>
+                <select name="x-ray1_ordered">  
```

Comment:
These aren't htmlform tags.  Can you update this @lebbiesahr ?

Reply:
yes, this code was added to let you see what i wanted to put in - i don't know if i should add these lists for x-ray/ultrasound as they are i.e. local to SL implementation or try to make part of wider list.  I think they are specific sub-lists so could keep local.  Will amend the tags once approach agreed.

Reply:
@lebbiesahr  I would recommend that you add "real" code but comment it out until the concepts are available.  For example,

<!-- ToDo: UHM-1234 Model radiology concepts and update 
<obs concept_id="PIH:foo" />
-->